spec/features/user_edits_account_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+feature 'user edits account', %Q{
+
+  As a registered user
+  I want to be able to edit my account
+  So my information is up to date
+  } do
+    #Acceptance Criteria
+    # *I must be logged in
+    # *I can only edit my own account
+    # *I can change or upload an optional avatar
+    # *I can change my password
+    # *I can change my email
+    # *I cannot change my username
+
+    scenario 'authenticated user changes email, password, & avatar' do
+
+      user = FactoryGirl.create(:user)
+
+      visit new_user_session_path
+
+      fill_in 'Login', with: user.username
+      fill_in 'Password', with: user.password
+
+      click_button 'Log in'
+
+      visit edit_user_registration_path
+
+      fill_in 'Email', with: "test@test.com"
+      fill_in 'Password', with: "Testing12345"
+      fill_in 'Password confirmation', with: "Testing12345"
+      fill_in 'Current password', with: user.password
+
+      click_button 'Update'
+
+      expect(page).to have_content('Your account has been updated successfully.')
```

Comment:
Line is too long. [81/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.